docs/framework/migration-guide/net-compatibility-diagnostics.md

```diff
@@ -460,7 +460,7 @@ The .NET Compatibility Diagnostics are Roslyn-powered analyzers that help identi
 |Scope|Edge|  
 |Version|4.5|  
 |Type|Runtime|  
-|Affected APIs|<xref:System.Linq.Enumerable.Empty%60%601?displayProperty=fullName>|  
+|Affected APIs|<xref:System.Linq.Enumerable.Empty%60%601%2A?displayProperty=fullName>|  
```

Comment:
although the link now works, it seems to be an issue with the build system. why this method requires the *? I see that the page shows ms.assetid with the * but seems wrong somehow.